### DISCUSSION TITLE
Thankn You

### DISCUSSION THREAD

**Discussion by bmorelax** [lowcoder-org/lowcoder#132] [General]
Thank you for setting this up. I am new to all of this and this seems like it will be very useful. I was looking at Open Blocks and realized that they were no longer active and then saw the post on their issues board. Just wanted to say thank you to all who are involved in this.

**Comment by falnyr** [comment]
Thanks for the kind words! Please feel free to raise any issues or feedback either here or on our public [Discord](https://discord.gg/yw4y4bkZUy).
